KonyAccelerate/controllers/mobile/frmWifiInfoController/frmWifiInfoController.js

```diff
@@ -1,23 +1,78 @@
 define({
-
-
     /**
-    	* @function  formPreShowAction
+    	* @function  formPostShowAction
         * @description This function is invoked at the preshow of the form and checks 
         			whether the wifi info is fetched at the app init or not. 
         			If the wifi info is already fetched it will display the 
                     result or else it will fetch the wifi info 
         * @private
     */
-    formPreShowAction: function() {
-      let accelerateEventData = kony.store.getItem("accelerateEventData");
-      if(!kony.sdk.isNullOrUndefined(accelerateEventData)){
-        let wifiInformation = accelerateEventData.wifiInformation;
-        let wifiUserName = (wifiInformation.hasOwnProperty("wifi_name")) ? wifiInformation.wifi_name : "";
-        let wifiPassword = (wifiInformation.hasOwnProperty("wifi_password")) ? wifiInformation.wifi_password : "";
-        this.view.lblNetworkName.text = wifiUserName;
-        this.view.lblPasswordValue.text = wifiPassword;
-      }
+    formPostShowAction: function() {
+        let accelerateEventData = kony.store.getItem("accelerateEventData");
+        if (!kony.sdk.isNullOrUndefined(accelerateEventData)) {
+            let wifiInformation = accelerateEventData.wifiInformation;
+            let wifiUserName = (wifiInformation.hasOwnProperty("wifi_name")) ? wifiInformation.wifi_name : "";
+            let wifiPassword = (wifiInformation.hasOwnProperty("wifi_password")) ? wifiInformation.wifi_password : "";
+            this.view.lblNetworkName.text = wifiUserName;
+            this.view.lblPasswordValue.text = wifiPassword;
+          	this.view.flxWifiImageContainer.isVisible = true;
+          	this.animateWifiCard("62%");
+        } else {
+            fetchObjectData(eventConstants.OBJECT_SERVICE_NAME, eventConstants.WIFI_OBJECT_NAME, {}, this.wifiFetchSuccess.bind(this), this.wifiFetchFailure.bind(this));
+        }
+      	
     },
+    /**
+     * @function wifiFetchSuccess
+     * @description This function is invoked in the success of wifi object and used to set the data to form
+     * @param successResponse The success response of the wifi object
+     * @private
+     */
+    wifiFetchSuccess: function(successResponse) {
```

Comment:
Can it be ```wifiServiceSuccess```?

Reply:
Actually we are using the same naming convention for object service.
"[ObjectName][Operation][Success/Failure]"